website/mailinglists/services.py

```diff
@@ -163,7 +163,7 @@ def get_automatic_lists():
                 f"to send newsletters in {language[1]}",
                 "addresses": _get_members_email_addresses(
                     Member.current_members.all().filter(
-                        profile__receive_newsletter=True, profile__language=language[0]
```

Comment:
This line is technically required to actually consider this functionality for each language. So either:
1. This becomes a separate issue (my preference)
2. Is also changed

Reply:
I don't really get what you're saying. What is required?

Reply:
We just need 1 mailinglist without the language in it. Thus the for-loop should also be removed.